hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/domain/InboundResponseFromCountry.java

```diff
@@ -10,6 +12,10 @@
     private Optional<String> statusMessage;
     private Optional<String> encryptedIdentityAssertionBlob;
     private Optional<LevelOfAssurance> levelOfAssurance;
+    @JsonProperty
```

Comment:
I'm unsure why this boolean property alone has to be annotated as a `@JsonProperty`. Could it be removed if we renamed the property to`assertionsUnsigned`?

Reply:
I think it's more down to the getter's name than the property's; I may have been able to annotate the getter instead of the property but it serves as a better warning sticking out at the top.

Because of the dark magic used in the bowels of Jackson (or whatever library it is), the getter will be ignored unless it's called `isAssertionsUnsigned` and given that possibility, I'd rather take the annotation - a) because it sounds ugly in my head when I read it and b) because there's a chance it could be misread as `isAssertionUnsigned` which is a thing in verify-saml-libs that's used in a stream operation to set the property ...

It's all very confusing but the take home is it took me a lot longer than I'd hope it would to smoke the "bug" out!